.github/workflows/main.yml

```
@@ -1,106 +1,61 @@
 
-env:
-  INSTANCE: 'Writerside/cb'
-  ARTIFACT: 'webHelpHI2-all.zip'
-  DOCKER_VERSION: '233.14938'
-
 jobs:
-  deploy:
+  checkout:
     runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get_version.outputs.VERSION }}
```

actionlint (1.7.7)

property "get_version" is not defined in object type {}

(expression)